src/sardana/pool/poolonedexpchannel.py

```diff
@@ -66,3 +67,34 @@ def read_data_source(self):
 
     data_source = property(get_data_source,
                            doc="source identifier for the 1D data")
+
```

Comment:
What about adding a `PoolTimerableChannel` in `sardana.pool.poolbasechannel`.
It would iherit from `PoolBaseChannel` and would be a base class for `PoolCounterTimer`, `Pool1DExpChannel` 
 and `Pool2DExpChannel`. This way we eliminate the code duplication.